utils/generate_invite_url.py

```diff
@@ -1,128 +1,44 @@
 """Utility function to generate the URL to invite the bot to a given Discord guild."""
 
 
-from collections.abc import Sequence
-
-__all__: Sequence[str] = ("InviteURLGenerator",)
-
-import os
 import re
-import sys
-from argparse import Namespace
-from typing import TYPE_CHECKING, Final
+from collections.abc import Sequence
```

Comment:
The module object exports should be placed at the top of the file. This line should be below the file's docstring but above every other line.